frontend/src/components/SkillGroup/SkillItem/styled.ts

```diff
@@ -38,9 +38,7 @@ export const SkillName = styled('p')({
   fontWeight: '400',
   fontSize: '18px',
   lineHeight: 1.2,
-  whiteSpace: 'nowrap',
-  overflow: 'hidden',
-  textOverflow: 'ellipsis',
+  textAlign: 'center',
```

Comment:
Why you deleted this? You tested it? I think, that without this properties we will have ui bag. Text overflow from container.